weed/server/master_server.go

```diff
@@ -50,11 +50,11 @@ func NewMasterServer(r *mux.Router, port int, metaFolder string,
 
 	var preallocateSize int64
 	if preallocate {
-		preallocateSize = int64(volumeSizeLimitMB) * (1 << 20)
+		preallocateSize = int64(volumeSizeLimit) * (1 << 20)
```

Comment:
This is wrong.

Reply:
yes, there is no need to  * (1 << 20) again.

Reply:
but the preallocateSize is not implemented yet, right?

Reply:
which part would you like to merge? I will pr the part which you will accept.